### ISSUE TITLE
"Bin total width" image incorrectly shows clearance between bottom profiles

### ISSUE THREAD

**Issue by pgroon** [Stu142/Gridfinity-Documentation#3]
The file "[bin total width](https://github.com/Stu142/Gridfinity-Documentation/blob/main/drawing_svg/bin_total_width.svg)" shows a .5mm clearance between the two bottom profiles on a 2U width bin. Zack's original models don't have this clearance, the two chamfers meet up at a 90° angle to each other.
![2024-04-29_21-04](https://github.com/Stu142/Gridfinity-Documentation/assets/46387880/5931d0de-6ce0-43df-90d4-5deda6f164e8)
![2024-04-29_21-05](https://github.com/Stu142/Gridfinity-Documentation/assets/46387880/5ea27227-6861-47d8-9f86-c81783fa8426)

I'm not sure where this comes from, since otherwise the depicted model seems spot on in each and every dimension listed.

**Comment by Stu142** [comment]
You are correct, I forgot about this. This was an intentional modification when I made my model a long time ago. I believe it is better than the original, but it's probably good to show both at the minimum, given I am claiming to show the original spec. 

You can see in the image below of zachs original model it does something funky at the meeting point. My model is where the red line would cut it off and make it even. 


![Screenshot 2024-04-29 175731](https://github.com/Stu142/Gridfinity-Documentation/assets/74122014/8a630337-6669-42ef-8137-5a1784dc9a5e)

I will try and add this when I get the chance
